e2e/k8s_test.go

```diff
@@ -156,3 +160,46 @@ func TestLargeDecodedVariablesSecretProvidedK8s(t *testing.T) {
 
 	testenv.Test(t, f.Feature())
 }
+
+*/
+
+func TestConjurSecretsNoPermissionK8s(t *testing.T) {
+	f := features.New("no permission to view conjur secrets").
+		// Replaces TEST_ID_12_no_conjur_secrets_permission
+		Assess("no permission to view conjur secrets in pod", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			// set environment variable in local environment
```

Comment:
this section of setting/getting env variables then reloading the template is not finished.

the environment variables being pulled seem to be pulled directly from my system, and not `bootstrap.env`, thus I am unable to configure a proper CONJUR_AUTHN_LOGIN id that expectedly causes the conjur secret retrieval to fail.